cli/ops/worker_host.rs

```diff
@@ -40,6 +51,21 @@ pub fn init(rt: &mut deno_core::JsRuntime) {
   );
   super::reg_json_sync(rt, "op_host_post_message", op_host_post_message);
   super::reg_json_async(rt, "op_host_get_message", op_host_get_message);
+  super::reg_json_sync(
+    rt,
+    "op_host_unhandled_error",
+    move |_state, args, _zero_copy| {
```

Comment:
Was there a reason to not be consistent with the other ops and create a seperate function?

Reply:
It's a closure that uses structures only safe (or convenient) to retrieve during init, not available in op context under current infrastructure. Suggested here: https://discord.com/channels/684898665143206084/684911491035430919/766354819925278750.